### PR TITLE
Added formData to Options interface

### PR DESCRIPTION
**Pull request by shiyala**
Fix issue #5787

### CODE REVIEW

File: request/request.d.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ declare module 'request' {
 			uri?: string;
 			callback?: (error: any, response: http.IncomingMessage, body: any) => void;
 			jar?: any; // CookieJar
+			formData: any; // Object
```
Comment (vvakame) [inline request/request.d.ts]: formData is required always? not optional?

Comment (Shiak1) [inline request/request.d.ts]: Actually you're right, wanna change it and patch it in?

**Comment by vvakame** [comment]
thanks mate!

**Comment by Shiak1** [comment]
Thanks for merging it

**Pull request merged**